examples/chart/teleport-kube-agent/.lint/db.yaml

```diff
@@ -1,3 +1,4 @@
+teleportVersionOverride: "10.0.0" # workaround the db section not rendering on pre-release versions
```

Comment:
```suggestion
```

As version 6 is long since end-of-life now, I'd say we can remove the lines in `config.yaml` which gate the `db_service` functionality and check the Teleport version (https://github.com/gravitational/teleport/blob/master/examples/chart/teleport-kube-agent/templates/config.yaml#L60-L61) which will mean we don't need to override the version here.